bedrock/firefox/templates/firefox/developer.html

```diff
@@ -48,23 +39,19 @@
 <section class="intro container">
 
   <h1>{{ _('Built for those who build the Web') }}</h1>
-  <h2>
-  {% if l10n_has_tag('spring_release') %}
-    {{ _('The only browser made for developers like you.') }}
-  {%- else -%}
-    {{ _('Introducing the only browser made for developers like you.') }}
-  {% endif -%}
-  </h2>
+  <h2>{{ _('The only browser made for developers like you.') }}</h2>
   {{ download_firefox('alpha', icon=False, small=True, simple=True) }}
 
-  {% if l10n_has_tag('new_doorhanger') %}
   <p class="feedback-note">
     {{ _('Firefox Developer Edition automatically sends feedback to Mozilla.') }}
     <a href="{{ url('privacy.notices.firefox') }}#telemetry" class="more">{{ _('Learn more') }}</a>
   </p>
-  {% endif %}
 
-  <img src="{{ static('img/firefox/developer/screenshot.jpg') }}" alt="{{ _('Firefox Developer Edition') }}" class="screenshot" />
+  {% if l10n_has_tag('firefox_dev_fall') and waffle.switch('firefox-dev-fall') %}
+    {{ high_res_img('firefox/developer/challenger-hero.png', {'alt': _('Firefox Developer Edition'), 'width': '832', 'height': 'auto', 'class': 'screenshot'}) }}
```

Comment:
Hm, just noting we're sending a 156KB image down to mobile users. Not a blocker, but maybe a good place to try out the fancy new `srcset` and `sizes` attributes after fall campaign?